app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Kindly explicitly specify the name of the associated model's class. By default, Rails assumes that the associated model's name is the same as the name of the association. However, there may be cases where the associated model's name is different from the default naming convention. In such cases, you can use the class_name option to explicitly specify the name of the associated model's class.